### PR TITLE
Switch to using qpid bind

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
Relies on https://github.com/Katello/puppet-qpid/pull/52

File: manifests/qpid.pp [hunk 1]
```diff
@@ -25,8 +25,9 @@
     require   => Service['qpidd'],
     logoutput => true,
   } ~>
-  qpid::bind_event { ['entitlement.created', 'entitlement.deleted', 'pool.created', 'pool.deleted', 'compliance.created']:
+  qpid::bind { ['entitlement.created', 'entitlement.deleted', 'pool.created', 'pool.deleted', 'compliance.created']:
     ssl_cert => $client_cert,
     queue    => $candlepin_event_queue,
+    exchange => 'event',
```
Comment (ekohl) [inline manifests/qpid.pp]: You could introduce a ```$candlepin_event_exchange``` and hardcode the default there. That at least makes it easier to override later on.

**Comment by ehelms** [comment]
Updated to add a variable for the candlepin exchange as a placeholder

File: manifests/qpid.pp [hunk 2]
```diff
@@ -25,8 +26,9 @@
     require   => Service['qpidd'],
     logoutput => true,
   } ~>
-  qpid::bind_event { ['entitlement.created', 'entitlement.deleted', 'pool.created', 'pool.deleted', 'compliance.created']:
+  qpid::bind { ['entitlement.created', 'entitlement.deleted', 'pool.created', 'pool.deleted', 'compliance.created']:
```
Comment (ekohl) [inline manifests/qpid.pp]: This should now also be ```qpid::config::bind```

**Comment by ehelms** [comment]
Refactored!

File: manifests/qpid.pp [hunk 3]
```diff
@@ -2,31 +2,30 @@
...
-    path      => '/usr/bin',
-    require   => Service['qpidd'],
-    logoutput => true,
-  } ~>
-  qpid::bind_event { ['entitlement.created', 'entitlement.deleted', 'pool.created', 'pool.deleted', 'compliance.created']:
+  qpid::config_cmd { 'create katello entitlements queue':
+    command  => "add queue ${candlepin_event_queue} --durable",
+    unless   => "queues ${candlepin_event_queue}",
     ssl_cert => $client_cert,
+  } ~>
```
Comment (ekohl) [inline manifests/qpid.pp]: This can be a simple ```->``` I think.

**Comment by ehelms** [comment]
Updated per comments

**Comment by ekohl** [comment]
Since https://github.com/Katello/puppet-qpid/pull/52 is now merged, could you rebase to see if the tests pass?

**Comment by ehelms** [comment]
@ekohl that did the trick, thanks.

**Pull request merged**